backend/httpd/main.go

```diff
@@ -9,5 +9,8 @@ func main() {
 			"message": "pong",
 		})
 	})
-	r.Run() // listen and serve on 0.0.0.0:8080 (for windows "localhost:8080")
+	err := r.Run() // listen and serve on 0.0.0.0:8080 (for windows "localhost:8080")
```

Comment:
this comment tripped me up since it was actually being served on port 3001

Reply:
oops, fixed!